ros2launch/ros2launch/api/api.py

```diff
@@ -154,8 +163,19 @@ def launch_a_launch_file(*, launch_file_path, launch_file_arguments, debug=False
             launch_arguments=parsed_launch_arguments,
         ),
     ])
+    tails = {}
+    for name in sorted(option_extensions.keys()):
+        result = option_extensions[name].prelaunch(
+            launch_description,
+            args
+        )
+        launch_description = result[0]
+        if len(result) > 1:
+            tails[name] = result[1:]
```

Comment:
Is this variable used solely for debugging? It isn't ever read from.

Reply:
Yes, it was. Fixed in https://github.com/ros2/launch_ros/pull/216/commits/22d14af0b1bcd5f969531e210713df93db619046